sites/docs/pages/+layout.svelte

```diff
@@ -1,25 +1,26 @@
-<head>
-    <script async defer src="https://scripts.simpleanalyticscdn.com/latest.js"></script>
-</head>
-
 <script>
 	import '@evidence-dev/tailwind/fonts.css';
 	import '../app.css';
 	import { EvidenceDefaultLayout } from '@evidence-dev/core-components';
 	export let data;
 </script>
 
-<EvidenceDefaultLayout {data} 
-	githubRepo=https://github.com/evidence-dev/evidence
+<head>
+	<script async defer src="https://scripts.simpleanalyticscdn.com/latest.js"></script>
+</head>
+
+<EvidenceDefaultLayout
+	{data}
+	githubRepo="https://github.com/evidence-dev/evidence"
 	slackCommunity="https://slack.evidence.dev"
 	xProfile="https://twitter.com/evidence_dev"
 	builtWithEvidence
-	maxWidth="1500" 
+	neverShowQueries
```

Comment:
until we have a good proposal for a query / component / source code viewer, I'd advocate for keeping this available?